workbench/planning/reporting.py

```diff
@@ -49,8 +49,8 @@ def __init__(self, *, weeks, users=None):
         self._absences = defaultdict(lambda: [[] for i in weeks])
         self._milestones = defaultdict(lambda: defaultdict(list))
 
-    def add_planned_work(self, queryset):
-        for pw in queryset.filter(weeks__overlap=self.weeks).select_related(
+    def add_planned_work_and_milestones(self, planned_work_qs, milestones_qs=False):
```

Comment:
```suggestion
    def add_planned_work_and_milestones(self, planned_work_qs, milestones_qs=None):
```

Detail :) kommt in etwa aufs gleiche raus, aber `None` kommuniziert etwas besser, wofür es ist als `False`.

Reply:
Bzw, führst Du irgendwo `add_planned_work_and_milestones` aus, ohne das zweite Argument anzugeben? Falls nicht, evtl. das `=False` bzw. `=None` einfach weglassen?

Reply:
Eigentlich nicht. Ich war mir nicht sicher, ob eine Milestones-Auflistung in der Teamansicht Sinn macht, wenn keine geplanten Stunden im Zeitraum sind. Daher ist der Parameter mal optional. Man weiss halt nicht recht, warum das Projekt aufgeführt ist (tatsächlich ist wird es aufgeführt, weil einer im Team mal geplante Stunden auf dem Projekt hatte oder haben wird und es einen Milestone gibt).